COPY/usr/bin/manageiq-db-ready

```diff
@@ -13,7 +13,9 @@ end
 
 def database_ready?(db_yaml, env = "production")
   host, port = db_yaml[env].values_at("host", "port")
-  if service_ready?(host, port || 5432)
+  host ||= "localhost"
+  port ||= 5432
+  if service_ready?(host, port)
```

Comment:
Note, this also fixes an issue where the logging of the `port` was incorrect below on line 17/20 before if we defaulted to 5432.